src/autogluon/bench/eval/evaluation/preprocess/preprocess_openml.py

```diff
@@ -58,7 +59,9 @@ def preprocess_openml_input(
 
     cleaned_input = preprocess_utils.clean_result(raw_input, folds_to_keep=folds_to_keep, remove_invalid=False)
 
-    cleaned_input["tid"] = [int(x.split("/")[-1]) for x in cleaned_input["id"]]
+    cleaned_input["tid"] = [
+        int(part) for x in cleaned_input["id"] for part in re.split(r'[/_]', x)[-1:] if part.isdigit()
+    ]
```

Comment:
Is it a new dataset naming format that we use for timeseries?

Reply:
task names in timeseries use underscores rather than slashes